bin/silence

```diff
@@ -37,9 +37,10 @@ Usage:
        --expiration:   silence expiration time, minutes. The silence stops at the time calculated as
                        the start time plus expiration time. 
        --key:          alert key to match
-       --var_name:     variable name to match
+       --var_name:     variable name. Accepts regular expression.
        --dev_id:       device Id to match
-       --dev_name:     device name
+       --dev_name:     device name. Accepts regular expression.
+       --comp_name:    component name. Accepts regular expression.
        --index:        component index to match
```

Comment:
please add a caveat to make it explicit that --index value can only be a single number